bliss/predict.py

```diff
@@ -77,7 +77,7 @@ def predict_on_image(
     assert image_encoder.border_padding == galaxy_encoder.border_padding
     assert image_encoder.tile_slen == galaxy_encoder.tile_slen
 
-    galaxy_param_mean = galaxy_encoder(ptiles, tile_map["locs"])
+    galaxy_param_mean, _ = galaxy_encoder(ptiles, tile_map["locs"])
```

Comment:
Related question: does the galaxy_encoder now return a `galaxy_param_mean` and `galaxy_param_var` or am I missing something?

Reply:
Right now galaxy_encoder returns a single sample from the variational distribution (and the associated density ratio of that sample) to minimize changes to the rest of the code. I'll likely refactor this in #363 by having the encoder forward method return a "Normal" object (which can be sampled, have the mean and var extracted, etc.)

Reply:
Since this isn't the mean, I'll rename it

Reply:
Sounds good, thanks for clarifying. I think some of the questions I had about these changes now make a lot more sense and we can merge once we test on real data.